src/common/KokkosKernels_BitUtils.hpp

```diff
@@ -46,6 +46,10 @@
 #define _KOKKOSKERNELS_BITUTILS_HPP
 #include "Kokkos_Core.hpp"
 
+#if defined (_MSC_VER)
```

Comment:
I'm wondering if `_MSC_VER` should be changed to the Kokkos macro `KOKKOS_COMPILER_MSVC` from here:
https://github.com/kokkos/kokkos/blob/98b8daf4a5644c3166123ded7bfa5b4d0264399b/core/src/Kokkos_Macros.hpp#L190 

@lucbv @brian-kelley do you have a preference on macro here?

Reply:
Let us use the Kokkos version for consistency